README.md

```diff
@@ -79,13 +79,14 @@ color: var(--rh-color-brand, #ee0000);
 They also provide reverse lookup, so if you want to choose between _all_ the tokens with value `#e00`, you can do so by completing the prefix `e00`.
 
 Load snippets in **VSCode**:
-Search for `Red Hat Design System` in the VSCode marketplace.
+Search for `Red Hat Design System` in the VSCode marketplace, or download the VSIX bundle from
+[the releases page](https://github.com/redhat-ux/red-hat-design-tokens/releases).
```

Comment:
Wouldn't the extension marketplace search be something more like `@category:"snippets" "Red Hat Design Tokens"` instead of searching `Red Hat Design System`? 

This given the content of `editor/vscode/package.json`

```
  "displayName": "Red Hat Design Tokens",
  "categories": [
    "Snippets"
  ],
  "contributes": {
    "snippets": [
      {
        "language": "css",
        "path": "./snippets.json"
      },
      {
        "language": "scss",
        "path": "./snippets.json"
      }
    ]
  },
  ```
  
  Maybe the more generic search would work given the description field: `Red Hat Design System Tokens`?

Reply:
:shrug: probably? make any changes here you see fit